pmlc/util/enums.h

```diff
@@ -58,4 +59,27 @@ inline DataType promoteTypes(DataType lhs, DataType rhs) {  //
   return std::max(lhs, rhs);
 }
 
+inline DataType from_string(const std::string& dtype) {
```

Comment:
I'm not sure why we need any of this. Tablegen generates the `symbolizeDataType` function for the purpose of converting strings into `DataType`.